tutorials/spanruler_restaurant_reviews/configs/ner_ruler.cfg

```diff
@@ -0,0 +1,47 @@
+[paths]
+train = null
+dev = null
+vectors = "en_core_web_lg"
+init_tok2vec = null
+
+[system]
+gpu_allocator = null
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec","ner", "span_ruler"]
```

Comment:
Oh wait, is this correct? I thought you said to attach the `SpanRuler` component before the `ner` component, which would also make more sense to me?

Reply:
I've now read some of the other discussions on this thread. Have we experimented with both approaches, compared F-scores between having `span_ruler`  before or after the `ner`?

Reply:
I ran a test before and it showed that running the `span_ruler` after `ner` works better (and it seems more consistent with what we want to showcase). However, I did that before adding a few more rules this year (after the holidays). I'll rerun it again and double-check to be sure.

Reply:
Reran the pipeline with the `span_ruler` before and it's still not as performant as adding it `after`. Will keep it the same way for now